segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/DurableLogConfig.java

```diff
@@ -32,6 +32,10 @@
     public static final Property<Integer> CHECKPOINT_COMMIT_COUNT = Property.named("checkpoint.commit.threshold.count", 300, "checkpointCommitCountThreshold");
     public static final Property<Long> CHECKPOINT_TOTAL_COMMIT_LENGTH = Property.named("checkpoint.commit.length.total", 256 * 1024 * 1024L, "checkpointTotalCommitLengthThreshold");
     public static final Property<Integer> START_RETRY_DELAY_MILLIS = Property.named("start.retry.delay.millis", 60 * 1000, "startRetryDelayMillis");
+    public static final Property<Integer> MAX_BATCHING_DELAY_MILLIS = Property.named("throttler.max.batching.delay.millis", ThrottlerCalculator.MAX_BATCHING_DELAY_MILLIS);
```

Comment:
Let's move these constants to its own class `ThrottlerConfig`.
Component name is `throttler`
These are not only related to Durable log

Reply:
Answered above (throttling is a subcomponent of `DurableLog` and is configured there, as metadata checkpoints).